File: 프로그래머스/lv1/42748. K번째수/K번째수.js
function solution(array, commands) {
 let newArray = []
  commands.map((v) => {
      let i = v[0]
      let j = v[1]
      let k = v[2]
      
      newArray.push(array.slice(i-1, j).sort((a,b) => a - b)[k-1])
    })
    
    return newArray
}

File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    
    for command in commands:
        start = command[0]
        end = command[1]
        target = command[2]
        
        aa = array[start - 1 : end]
        aa.sort()
        
        answer.append(aa[target - 1])
        
    return answer
        
        